### PR TITLE
Added soot java9 configuration with demo projects

### PR DESCRIPTION
**Pull request by rakshitkr**

### CODE REVIEW

**Comment by johspaeth** [comment]
Thanks @rakshitkr could you please have a look at why the CI fails?

**Comment by rakshitkr** [comment]
> Thanks @rakshitkr could you please have a look at why the CI fails?

Yes... Will look into it

File: CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java [hunk 1]
```diff
@@ -288,8 +288,28 @@ private void initializeSootWithEntryPointAllReachable(boolean wholeProgram) {
...
+			System.out.println("java 9 classpath settings applied");
+			Options.v().set_soot_classpath("VIRTUAL_FS_FOR_JDK" + File.pathSeparator + sootClassPath());
+		}
+		
+//		// JAVA 9 && MODULEPATH
+//		{
+//			System.out.println("java 9 modulepath settings applied");
+//			Options.v().set_prepend_classpath(true);
+//			Options.v().set_soot_modulepath(sootClassPath());
+//		}
```
[the remark this answers is not in the document]
Comment (rakshitkr) [reply CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java]: I've used this code in the latest version. So its not longer commented

File: CryptoAnalysis/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>de.fraunhofer.iem</groupId>
 	<artifactId>CryptoAnalysis</artifactId>
-	<version>2.4-SNAPSHOT</version>
+	<version>4.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (rakshitkr) [reply CryptoAnalysis/pom.xml]: I had changed for testing purpose to ensure that CogniCrypt uses correct CryptoAnalysis jar.

**Pull request merged**